static/app/views/replays/detail/detailLayout.tsx

```diff
@@ -70,6 +100,7 @@ function DetailLayout({children, event, orgId, crumbs}: Props) {
           </React.Fragment>
         </Layout.Header>
         {children}
+        <IndicatorContainer />
```

Comment:
This shouldn't be necessary, it should already be rendered in our global layout.